[Bundle] Sync doc pages with current Symfony codebase (#19185)
symfony/symfony-docs: bundles.rst

    return [
        // 'all' means that the bundle is enabled for any Symfony environment
        Symfony\Bundle\FrameworkBundle\FrameworkBundle::class => ['all' => true],
        Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
        Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
        // this bundle is enabled only in 'dev'
        Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true],
        Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
        // this bundle is enabled only in 'dev' and 'test', so you can't use it in 'prod'
        Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
        Twig\Extra\TwigExtraBundle\TwigExtraBundle::class => ['all' => true],
        Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
        Symfony\Bundle\MonologBundle\MonologBundle::class => ['all' => true],

>         Symfony\Bundle\MakerBundle\MakerBundle::class => ['dev' => true],

Reviewer: Lets sort the alpabetically

Author: My goal there was to present this file as it is after a standard install (—webapp) to avoid confusion to newcomers

Reviewer 2: I would make this more focused on what we actually want to show here:

```suggestion
        // ...

        // this bundle is enabled only in 'dev'
        Symfony\Bundle\DebugBundle\DebugBundle::class => ['dev' => true],
        // ...

        // this bundle is enabled only in 'dev' and 'test', so you can't use it in 'prod'
        Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
        // ...
```

between all Symfony bundles. It follows a set of conventions, but is flexible
to be adjusted if needed:

``assets/``
    Contains the web asset sources (JavaScript and TypeScript files, CSS and Sass
    files, etc.), images and other assets related to the bundle that are not in

>     ``public/`` (e.g. Stimulus controllers)

Reviewer 3: ```suggestion
    ``public/`` (e.g. Stimulus controllers).
```

>     Contains the web asset sources (JavaScript and TypeScript files, CSS and Sass
>     files, etc.), images and other assets related to the bundle that are not in
>     ``public/`` (e.g. Stimulus controllers)

Reviewer 3: Also, you may use a list without parentheses?

```suggestion
    Contains the web asset sources like JavaScript and TypeScript files, CSS and Sass
    files, but also images and other assets related to the bundle that are not in
    ``public/`` (e.g. Stimulus controllers).
```

``config/``

>     Houses configuration, including routing configuration (e.g. ``routing.yaml``).

Reviewer 4: Wondering if we should set `routes.php` instead? I seems more common for bundle nowday

Author: Well i just went through 50% of the bundles listed on the symfony.com dedicated section .... 

... and did not find a single config/routing.php. ... but i did not find more config/routing.yaml ... 

Almost all those bundle are either:
* still on the "old" structure
* not using config files

So i'd say it's even more a good move..  
Because if no one uses/shows this new structure (with routes)... no bundle will follow the trend :)

Author: Commited
